tests/framework/web/UrlRuleTest.php

```diff
@@ -513,6 +513,20 @@ protected function getTestsForCreateUrl()
                 ],
             ],
             [
+                'single optional param with suffix',
+                [
+                    'pattern' => '<page>',
+                    'route' => 'page/view',
+                    'defaults' => ['page' => 'index'],
+                    'suffix' => '/',
+                ],
+                [
+                    ['page/view', [], '/'],
+                    ['page/view', ['page' => 'index'], '/'],
```

Comment:
The correct one is empty string.
After #13624 merge the result will be correct
See https://github.com/yiisoft/yii2/pull/13624/commits/7206b2ba8a9e2a7648fa5507ce338fc851bc0a6e#diff-96fb0ef29c09f20871090c41bf5b3839R498